src/components/input-number/input-number.scss

```diff
@@ -0,0 +1,448 @@
+  }
+  & input[readonly] {
+    @apply bg-background font-medium;
+  }
+  & input[readonly]:focus {
+    @apply text-color-1;
+  }
+  & calcite-icon {
+    @apply text-color-3;
+  }
+}
+
+//focus
+:host input {
+  @apply focus-base;
+}
+
+:host input:focus {
+  @apply focus-inset;
+}
+
+:host([status="invalid"]) {
+  & input {
+    @apply border-color-danger;
+  }
+  & input:focus {
+    @apply focus-inset-danger;
+  }
+}
+
+// ICONS
+
+// position icons
+
+:host([scale="s"]) .icon {
+  inset-inline-start: theme("spacing.2");
+}
+
+:host([scale="m"]) .icon {
+  inset-inline-start: theme("spacing.3");
+}
+
+:host([scale="l"]) .icon {
+  inset-inline-start: theme("spacing.4");
+}
+
+// position placeholder/value text in relation to icons
+
+:host([icon][scale="s"]) input {
+  padding-inline-start: theme("padding.8");
+}
+
+:host([icon][scale="m"]) input {
+  padding-inline-start: theme("padding.10");
+}
+
+:host([icon][scale="l"]) input {
+  padding-inline-start: theme("padding.12");
+}
+
+// positioning wrapper for icon and loader
+
+.element-wrapper {
+  @apply relative
+    order-3
+    inline-flex
+    flex-1
+    items-center;
+}
+
+.icon {
+  @apply transition-default
+    pointer-events-none
+    absolute
+    z-10
+    block;
+}
+
+.clear-button {
+  pointer-events: initial;
+  @apply focus-base
+    border-color-input
+    bg-foreground-1
+    order-4
+    m-0
+    box-border
+    flex
+    min-h-full
+    cursor-pointer
+    items-center
+    justify-center
+    self-stretch
+    border
+    border-solid;
+
+  border-inline-start-width: theme("borderWidth.0");
+
+  &:hover {
+    @apply bg-foreground-2 transition-default;
+    calcite-icon {
+      @apply text-color-1 transition-default;
+    }
+  }
+  &:active {
+    @apply bg-foreground-3;
+    calcite-icon {
+      @apply text-color-1;
+    }
+  }
+  &:focus {
+    @apply focus-inset;
+  }
+  &:disabled {
+    @apply opacity-disabled;
+  }
+}
+
+// loading
+.loader {
+  top: 1px;
+  left: 1px;
+  right: 1px;
+  @apply pointer-events-none
+    absolute
+    block;
+}
+
+// slotted action
+.action-wrapper {
+  @apply order-7 flex;
+}
+
+// prefix and suffix
+.prefix,
+.suffix {
+  @apply border-color-input
+    bg-background
+    text-color-2
+    box-border
+    flex
+    h-auto
+    min-h-full
+    select-none
+    content-center
+    items-center
+    break-words
+    border
+    border-solid
+    font-medium
+    leading-none;
+}
+
+.prefix {
+  @apply order-2;
+  border-inline-end-width: theme("borderWidth.0");
+}
+.suffix {
+  @apply order-5;
+  border-inline-start-width: theme("borderWidth.0");
+}
+
+// alignment type
+:host([alignment="start"]) {
+  & input {
+    text-align: start;
+  }
+}
+
+:host([alignment="end"]) {
+  & input {
+    text-align: end;
+  }
+}
+
+:host input {
+  -moz-appearance: textfield;
+
+  &::-webkit-inner-spin-button,
+  &::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    -moz-appearance: textfield;
+    @apply m-0;
+  }
+}
```

Comment:
This is probably safe to remove since we're not rendering a `type="number"` input element, and haven't been since we first added locale support.